webcompat/helpers.py

```diff
@@ -133,6 +133,15 @@ def get_name(dictionary):
     return name
 
 
+def get_browser_version(user_agent_string=None):
+    """Return browser version number as an integer."""
+    if user_agent_string and isinstance(user_agent_string, str):
+        ua_dict = user_agent_parser.Parse(user_agent_string)
+        version = int(ua_dict['user_agent']['major'])
```

Comment:
In theory it's possible for https://github.com/tobie/ua-parser/blob/master/py/ua_parser/user_agent_parser.py#L60 to return `None` (which populates the `major` key) however unlikely, so I think there's a chance to throw a TypeError when you try to convert `None` to `int`.

The default 0 return would solve this if you had a try/except clause.